.github/workflows/instrumentation-test.yml

```diff
@@ -1,11 +1,16 @@
 name: polyPod Android integration test
 on:
-    schedule:
-      - cron: '31 13 * * *'
+    push:
+        paths:
+            - "features/test/*"
+            - "platform/feature-api/**"
+            - "platform/core/**"
+            - "platform/android/**"
+            - ".github/workflows/instrumentation-test.yml"
```

Comment:
If we don't run these nightly, they should in theory run for _any_ changes that could break them, including changes to any (bundled) feature. Maybe just the same pattern as on iOS for now?

```
 - "feature-utils/**"
 - "features/**"
 - "platform/feature-api/**"
 - "platform/core/**"
 - "platform/ios/**"
```